invokeTarkovTextract/template.yaml

```
@@ -0,0 +1,14 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: 'AWS::Serverless-2016-10-31'
+Description: An AWS Serverless Specification template describing your function.
+Resources:
+  invokeTarkovTextract:
+    Type: 'AWS::Serverless::Function'
+    Properties:
+      FunctionName: invokeTarkovTextract
+      Description: ''
+      Handler: lambda_function.lambda_handler
+      MemorySize: 128
+      Role: 'arn:aws:iam::832044451952:role/TarkovLambdaRole'
```

Should we be committing ARNs?
```suggestion
      Role: 'arn:aws:iam::832044451952:role/TarkovLambdaRole'
```